<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ARM</class>
 <widget class="QMainWindow" name="ARM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ARM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnTeacher">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
    </property>
    <property name="text">
     <string>Я преподаватель</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeader">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
    </property>
    <property name="text">
     <string>Я лидер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStudent">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
    </property>
    <property name="text">
     <string>Я студент</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Вход в систему</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
